### DISCUSSION TITLE
Are multiple API versions possible with this package?

### DISCUSSION THREAD

**Discussion by keenminded** [binaryk/laravel-restify#507] [Q&A]
Hi, all!
Is it possible to have and maintain multiple API versions with this package? For example, with multiple prefixes? You would have routes /api/v1/*, /api/v1.1/*, etc.
It wasn't clear how to achieve this through the documentation.
I appreciate any help you can provide.

**Comment by arthurkirkosa** [comment]
I'd say no, at the moment.
